### PR TITLE
Removed unnecessary permissions from Manifest file

### PR DESCRIPTION
**Pull request by akashdabhi03**
|PR checklist|Status|
|-----|------|
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|Yes|
|Is code analysis task executed and issues resolved in the CPP/native code?|NA|
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|Yes|
|Is code commenting done properly for the changes made?|NA|
|Is correct variable naming convention followed?|NA|
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|Yes|
|Are the message strings used in the application taken from strings.xml file?|NA|
|Are the permissions which are not required removed from AndroidManifest.java file?|Yes|
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|Yes|
|Are the generated files correctly configured in .gitignore?|NA|
|Are the release signing configurations removed from the gradle files?|Yes|
|Is commented code removed from all files?|Yes|
|Are updated images compressed (< 100 KB)?|NA|
|Typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|No|

OBB Checklist:

|PR checklist|Status|
|-----|------|
|Is size of the OBB correct?|Yes|
|Is OBB file version correctly set in DownloadExpansionFile.java file?|Yes|
|Is patch OBB file to be deployed? If yes, then are the settings correct in DownloadExpansionFile.java file?|Yes|

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -38,96 +20,18 @@
...
-
-
-        <activity
-            android:name="com.maq.xprize.onecourse.hindi.mainui.SplashScreenActivity"
-            android:configChanges="orientation|screenSize|keyboardHidden"
-            android:launchMode="singleTask"
-            android:screenOrientation="sensorLandscape">
-        </activity>
+        android:theme="@style/AppTheme"
+        tools:ignore="GoogleAppIndexingWarning">
```
Comment (ankitk20) [inline app/src/main/AndroidManifest.xml]: Remove ignore warning

File: app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java [hunk 2]
```diff
@@ -82,7 +80,7 @@
     public static OBConfigManager configManager;
     public static OBSystemsManager systemsManager;
     public static OBAnalyticsManager analyticsManager;
-    public static OBLocationManager locationManager;
+//    public static OBLocationManager locationManager;
```
Comment (ankitk20) [inline app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java]: Remove this if not required

File: app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java [hunk 3]
```diff
@@ -579,10 +568,11 @@ public boolean isMicrophonePermissionGranted() {
     public boolean isCameraPermissionGranted() {
         Boolean micPermission = selfPermissionGranted(Manifest.permission.CAMERA) == PackageManager.PERMISSION_GRANTED;
         //
-        if (!micPermission)
-            ActivityCompat.requestPermissions(this, PERMISSIONS_CAMERA, REQUEST_CAMERA);
+        /*if (!micPermission)
+            ActivityCompat.requestPermissions(this, PERMISSIONS_CAMERA, REQUEST_CAMERA);*/
         //
-        return micPermission;
+//        return micPermission;
```
Comment (ankitk20) [inline app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java]: Remove commented code

File: app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java [hunk 4]
```diff
@@ -186,21 +179,17 @@ public void uncaughtException(Thread paramThread, Throwable paramThrowable) {
...
         //getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
         //
         mainActivity = this;
         //
-        configManager = new OBConfigManager();
+        configManager = new OBConfigManager(this.getApplicationContext());
         //
         analyticsManager = new OBAnalyticsManager(this);
-        locationManager = new OBLocationManager(this);
+//        locationManager = new OBLocationManager(this);
```
Comment (ankitk20) [inline app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java]: Remove this

File: app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/SplashScreenActivity.java [hunk 5]
```diff
@@ -185,7 +177,14 @@ protected String doInBackground(String... sUrl) {
...
                 Toast.makeText(SplashScreenActivity.this, "Insufficient storage space! Please free up your storage to use this application.", Toast.LENGTH_LONG).show();
-                finish();
+
+                // Call finish after the toast message disappears
+                new Handler().postDelayed(new Runnable() {
+                    @Override
+                    public void run() {
+                        SplashScreenActivity.this.finish();
+                    }
+                }, 3500);
```
Comment (ankitk20) [inline app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/SplashScreenActivity.java]: Toast.LENGTH_LONG

File: app/src/main/java/com/maq/xprize/onecourse/hindi/utils/OBAnalyticsManagerCommunity.java [hunk 6]
```diff
@@ -180,7 +181,8 @@ public void deviceGpsLocation ()
     {
         Map<String, Object> parameters = new HashMap();
         //
-        Location loc = OBLocationManager.sharedManager.getLastKnownLocation();
+//        Location loc = OBLocationManager.sharedManager.getLastKnownLocation();
```
Comment (ankitk20) [inline app/src/main/java/com/maq/xprize/onecourse/hindi/utils/OBAnalyticsManagerCommunity.java]: Remove commented code from everywhere

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (akashdabhi03) [reply app/src/main/AndroidManifest.xml]: Correct

File: app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/SplashScreenActivity.java (hunk 5 above)
Comment (akashdabhi03) [reply app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/SplashScreenActivity.java]: Updated

File: app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java (hunk 2 above)
Comment (akashdabhi03) [reply app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java]: Removed

**Review by Shailin — DISMISSED** [review]

**Review by ankitk20 — DISMISSED** [review]

**Review by Shailin — DISMISSED** [review]

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**